book/src/05_storages.md

```diff
@@ -52,16 +52,16 @@ fairly well.
 
 ## `DenseVecStorage`
 
-This storage uses two `Vec`s, one containing the actual data and the other
+This storage uses two `Vec`\`s, one containing the actual data and the other
```

Comment:
Not a native speaker, but this change seems odd to me.

Reply:
Yeah this was fine before the change